exporter/datadogexporter/examples/collector.yaml

```diff
@@ -214,6 +214,27 @@ exporters:
       ## @param tls - boolean - optional - default: false
       # insecure_skip_verify: false
 
+    ## @param read_buffer_size - integer - optional
+    ## @param write_buffer_size - integer - optional
+    ## @param timeout - duration - optional
+    ## @param max_idle_conns - integer - optional
+    ## @param max_idle_conns_per_host - integer - optional
+    ## @param max_conns_per_host - integer - optional
+    ## @param idle_conn_timeout - duration - optional
+    ## @param disable_keep_alives - boolean - optional
+    ##
+    ## Below are a subset of configs in confighttp that are respected by Datadog exporter.
```

Comment:
I think we should return an error if the user tries to set an unsupported option instead of failing silently

Reply:
Makes sense, added a check